dcos/package.py

```diff
@@ -169,3 +169,34 @@ def merge_installed(apps, subcommands, app_only, cli_only):
         merged.extend(indexed_subcommands.values())
 
     return merged
+
+
+def get_cosmos_url():
+    """
+    :returns: cosmos base url
+    :rtype: str
+    """
+    toml_config = config.get_config()
+    cosmos_url = config.get_config_val("package.cosmos_url", toml_config)
+    if cosmos_url is None:
+        cosmos_url = config.get_config_val("core.dcos_url", toml_config)
+        if cosmos_url is None:
+            raise config.missing_config_exception(["core.dcos_url"])
+    return cosmos_url
+
+
+def get_package_manager():
+    """Returns type of package manager to use
+
+    :returns: PackageManager instance
+    :rtype: PackageManager
+    """
```

Comment:
Didn't we move this from `.../package/main.py` or something like that? Let's delete that code and change all of the call sites.